test/atomfs-erofs.bats

```diff
@@ -0,0 +1,164 @@
+load helpers
+
+function setup() {
+    stacker_setup
+}
+
+function teardown() {
+    cleanup
+}
+
+function verity_checkusedloops() {
+    # search for loopdevices which have backing files with the current
+    # BATS_TEST_DIRNAME value and complain if they're present.
+    local usedloops="" found="" x=""
+    for ((x=0; x<5; x++)); do
+        usedloops=$(losetup -a | grep $BATS_TEST_DIRNAME || echo)
+        if [ -n "$usedloops" ]; then
+            found=1
+            udevadm settle
+        else
+            return 0
+        fi
+    done
+    echo "found used loops in testdir=$BATS_TEST_DIRNAME :$usedloops" >&3
+    [ $found = 1 ]
+}
+
+function basic_test() {
+    require_privilege priv
+    local verity_arg=$1
+
+    cat > stacker.yaml <<"EOF"
+test:
+    from:
+        type: oci
+        url: ${{BUSYBOX_OCI}}
+    run: |
+        touch /hello
+EOF
+    stacker build --layer-type=erofs $verity_arg --substitute BUSYBOX_OCI=${BUSYBOX_OCI}
+    mkdir mountpoint
+    stacker internal-go atomfs mount test-erofs mountpoint
+
+    [ -f mountpoint/hello ]
+    stacker internal-go atomfs umount mountpoint
+}
+
+@test "--no-verity works" {
+    basic_test --no-verity
+    verity_checkusedloops
+}
+
+@test "mount + umount works" {
+    basic_test
+
+    # last layer shouldn't exist any more, since it is unique
+    manifest=$(cat oci/index.json | jq -r .manifests[0].digest | cut -f2 -d:)
+    last_layer_num=$(($(cat oci/blobs/sha256/$manifest | jq -r '.layers | length')-1))
+    last_layer_hash=$(cat oci/blobs/sha256/$manifest | jq -r .layers[$last_layer].digest | cut -f2 -d:)
```

Comment:
just a general comment, prolly worth an issue, but we can avoid the `cat | jq` and just `jq xxx < file`